client/src/components/survey/SurveyPage4.jsx

```diff
@@ -298,11 +291,11 @@ const SurveyPage4 = props => {
             </Button>
           </Grid>
         </Grid>
-        <div className={classes.chartDiv} />
       </div>
       <div />
-      <div className={classes.innerChartDiv} />
-      <div className={classes.tagLine} />
+      <div className={classes.innerChartDiv}>
+        <BehavioralChart />
+      </div>
```

Comment:
I feel like we should either include this only in `Table` or only in `SurveyPage4` but not both, as we don't have the temperature chart on `SurveyPage2` and don't want to inundate our everyday users with charts

Reply:
So I removed it from SurveyPage4. If it needs to be added, we can add it anytime in a future PR. So as of now the Behavioral Progress Chart is in Tables.jsx along with the Fever Chart.